sdk/instrumentation/net/http/handler.go

```diff
@@ -17,21 +19,31 @@ type handler struct {
 	defaultAttributes        map[string]string
 	spanFromContextRetriever sdk.SpanFromContext
 	dataCaptureConfig        *config.DataCapture
+	urlFilters               []filter.URLFilter
+	headersFilters           []filter.HeadersFilter
+	bodyFilters              []filter.BodyFilter
+}
+
+// Options for HTTP handler instrumentation
+type Options struct {
```

Comment:
I think it is better to put the filters inside a single field in options as options other options might come and then it will look very varied.

Reply:
Makes sense.